packages/venia-ui/lib/queries/getStoreConfigData.graphql

```diff
@@ -2,5 +2,10 @@ query storeConfigData {
     storeConfig {
         id
         copyright
+        code
+        locale
+        base_currency_code
+        default_display_currency_code
```

Comment:
You can remove the other properties from this query that aren't used.

Reply:
Done